src/anemoi/training/utils/checkpoint.py

```diff
@@ -65,3 +98,37 @@ def save_inference_checkpoint(model: torch.nn.Module, metadata: dict, save_path:
     torch.save(model, inference_filepath)
     save_metadata(inference_filepath, metadata)
     return inference_filepath
+
+
+def transfer_learning_loading(model: torch.nn.Module, ckpt_path: Path | str) -> nn.Module:
+
+    # Load the checkpoint
+    try:
+        checkpoint = torch.load(ckpt_path, map_location=model.device)
+
+    # TODO @anaprietonem: this is a patch for issue #57
+    except RuntimeError:
+        LOGGER.debug("Need to remove metadata from the checkpoint file due to issue #57..")
+
+        file_to_delete = "archive/anemoi-metadata/ai-models.json"
+        # Construct and execute the command
+        remove_file_from_zip(ckpt_path, file_to_delete)
```

Comment:
The side effects from deleting provenance information are quite significant. While we need to find a solution to the problem, modifying a checkpoint in-place and removing information from it cannot go into `develop` like this.

Reply:
My original implementation was creating a copy of the ckpt called "transfer.ckpt" where the provenance information is deleted. Should I re-implement this then?

Reply:
Patching the checkpoint directly in the code caused file access issues when running in a multi GPU setting.
If someone has ideas we can reintorduce this in the future.
For the moment, **I had to remove this** and simplify the code.
Nowthe code is expecting that the checkpoint has been already fixed previously as documented in #57, and is present in the directory with the name: {checkpoint_path}/{checkpoint_name}_patched.ckpt